### ISSUE TITLE
HOw to check correct column headers exist?

### ISSUE THREAD

**Issue by paulcanning** [thephpleague/csv#323]
Is there a way to check that certain column headers exist in a CSV being read?

I need to convert some CSVs to a different layout, and I need to make sure the original CSV has the correct headers, and if not, move to the next file.

**Comment by nyamsprod** [comment]
@paulcanning thanks for using the library. Have you read the documentation website ? [All information are there I believe](https://csv.thephpleague.com/9.0/reader/#csv-header)

**Comment by paulcanning** [comment]
@nyamsprod That doesn't answer my query.

I can get the header, but I need to check it has certain (column) header values.

**Comment by nyamsprod** [comment]
yes it does ... now that you have the reader you should be able to do what you want :+1: What you want to do with is is up to your dev skills :wink:
